python/postprocessing/analysis/higgs/vhbb/postproc.py

```diff
@@ -34,5 +36,5 @@
 Sum$(Jet_pt *(abs(Jet_eta)<2.5 && Jet_pt > 20 && Jet_jetId)) > 160  || 
 MET_pt > 100  || Sum$(Muon_pt > 20 && Muon_tightId) >= 1
 '''
-p=PostProcessor(".",files,selection.replace('\n',' '),"keep_and_drop.txt",[btagSFProducer("cmva"),jecUncertAll_cppOut(),vhbb()],provenance=True)
+p=PostProcessor(".",files,selection.replace('\n',' '),"keep_and_drop.txt",[jecUncertAll_cppOut(),jetmetUncertaintiesAll(),btagSFProducer("cmva"),vhbb()],provenance=True)
```

Comment:
@arizzi specifically I imagine it is this line that perhaps I am not doing correctly to include the JER

Reply:
it seems there is a jetSmearer module from @veelken can you try that?

Reply:
right the jetSmearer module is an integral part of this jetmetUncertainties module. My understanding from looking at the code for these modules is that one should call the jetmetUncertainties module which wraps around jetSmearer but maybe I have that wrong...